### PR TITLE
Basic test verifying if demo account is working properly

### PR DESCRIPTION
**Pull request by BOHEUS**

### CODE REVIEW

**Comment by BOHEUS** [comment]
It needs small fix as right now when running this test from CLI, it launches additional test used by other tests when it shouldn't as there's no need to save the state of page after authorization.

**Comment by FelixMalfait** [comment]
Hey @BOHEUS I'm not sure we want to check external urls in a test like that. I think tests like that are meant to be able to run in an isolated environment. Otherwise this could break for things not related to the code. Tests are meant to help prevent error during deployment/PR but this is more of a reactive alert type of thing which should be done some place else. If we want to test demo/prod urls there are other QA tools we could probably use.

**Comment by FelixMalfait** [comment]
I discussed with @charlesBochet and he doesn't agree. Could you try creating the github action so what what it'd look like? Maybe a cron running once a day that would send an email for example? Maybe we should add a Discord alert step https://github.com/marketplace/actions/action-notify

**Comment by BOHEUS** [comment]
No problem, I'll take care of it in free time, I'll focus on creating basic Github workflow with cron to run it once/twice per day and later I'll add additional features like email or Discord alert depending on needs

File: packages/twenty-e2e-testing/tests/all/companies.spec.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { test, expect } from '../lib/fixtures/screenshot';
+import { test, expect } from '../../lib/fixtures/screenshot';
 
 test.describe('Basic check', () => {
   test('Checking if table in Companies is visible', async ({ page }) => {
```
[the remark this answers is not in the document]
Comment (BOHEUS) [reply packages/twenty-e2e-testing/tests/all/companies.spec.ts:4]: There's initial step in login.setup.ts

File: packages/twenty-e2e-testing/tests/all/workspaces.spec.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { test, expect } from '@playwright/test';
-import { sh } from '../drivers/shell_driver';
+import { sh } from '../../drivers/shell_driver';
 
 test.describe('', () => {
```
[the remark this answers is not in the document]
Comment (BOHEUS) [reply packages/twenty-e2e-testing/tests/all/workspaces.spec.ts:4]: Not highest prio as this test requires refactor later in the future

**Comment by BOHEUS** [comment]
@charlesBochet I'm sorry I didn't inform earlier that this PR is ready to review, please check this workflow as I'm not entirely sure if that's a correct approach

**Review by charlesBochet — APPROVED** [review]
Sorry for the slow review, let's try it!

**Pull request merged**